nss-day-cohort-52/rare-api-droppedbagel
# Activate and Deactivate Users

Added custom actions to post list to alter the 'is_active' property on the django user model of a user.
Also modified list view to order users by username.

To Test in Postman:
- Perform a PUT request to a single user with the action "activate" and, if successful, the server will respond with the message "user has been set as active" and the relevant auth_user row in the database will be updated.
Ex: http://localhost:8000/users/2/activate

- Perform a PUT request to a single user with the action "deactivate" and, if successful, the server will respond with the message "user has been deactivated" and the relevant row in the database will be updated.
Ex: http://localhost:8000/users/2/deactivate

* * deactivated user tokens are no longer valid for authorization so test by deactivating an account other than the one you are using in your Postman authorization header.

Fixes #26 
Fixes #28 
Fixes #29
